hedera-node/hedera-smart-contract-service-impl/src/main/java/module-info.java

```diff
@@ -15,6 +16,7 @@
     requires transitive tuweni.bytes;
     requires transitive tuweni.units;
     requires com.hedera.node.app.service.evm;
+    requires com.github.benmanes.caffeine;
```

Comment:
This is new, do we actually use it?

Reply:
It is used in 

`hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/infra/EthTxSigsCache.java`

It is also used in `hedera-evm` and that's why it probably worked also without declaring it here.

Looking at the Git history, this seems to be a rather new addition.